hw06-atm-emulator/src/main/java/hw06/atm/Banknote.java

```diff
@@ -0,0 +1,38 @@
+package hw06.atm;
+
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
+public class Banknote {
+    private final static Set<Integer> POSIBLE_VALS = ImmutableSet.of(50, 100, 200, 500, 1000, 5000);
+    private final int cost;
+
+    public Banknote(int value){
+        if( !POSIBLE_VALS.contains(value)){
+            throw new RuntimeException("Undefined banknote type");
```

Comment:
Здесь правильно использовать `IllegalArgumentException` и еще передать в исключение, какая купюра была запрошена, чтобы проще было разобраться с ошибкой.

```suggestion
            throw new IllegalArgumentException("Undefined banknote type - " +value );
```

Подумайте еще над тем, чтобы использовать перечисление для номиналов.

Reply:
Про исключение понял. Исправлю.
Перечисления не стал использовать, так как у них концептуально нет метрики для сравнения (техническая  реализации не учитывается). Для работы с номиналами как с конечным численным множеством посчитал, что лучше подходит сет.

Reply:
На перечислениях не настаиваю, но, справедливости ради, надо отметить, что перечисления реализуют `Comparable`.

Можно здесь почитать немного об этом
https://vertex-academy.com/tutorials/ru/perechisleniya-enum-java-2/
https://stackoverflow.com/questions/519788/why-is-compareto-on-an-enum-final-in-java